server/pa.js

```diff
@@ -2,12 +2,37 @@
 const Validator = require('express-validator')
 
 // In-house dependencies
-const { helpers, twilioHelpers } = require('brave-alert-lib')
+const { helpers, twilioHelpers, googleHelpers } = require('brave-alert-lib')
 const db = require('./db/db')
 
 const paApiKeys = [helpers.getEnvVar('PA_API_KEY_PRIMARY'), helpers.getEnvVar('PA_API_KEY_SECONDARY')]
 const paPasswords = [helpers.getEnvVar('PA_PASSWORD_PRIMARY'), helpers.getEnvVar('PA_PASSWORD_SECONDARY')]
 
+const validateGetGoogleTokens = Validator.body(['authCode']).trim().notEmpty()
+
+async function getGoogleTokens(req, res) {
+  try {
```

Comment:
To use the validator to check that the request is valid, you need to do something like:
```
  const validationErrors = Validator.validationResult(req).formatWith(helpers.formatExpressValidationErrors)
  if (validationErrors.isEmpty()) {
```

at/near the top of the function. You can see it in the other API handler functions.